griptape/drivers/observability/base_observability_driver.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+from typing import Any
+
+
+class BaseObservabilityDriver(ABC):
+    def __enter__(self) -> None:
+        pass
+
+    def __exit__(self, exc_type, exc_value, traceback) -> None:
+        pass
```

Comment:
What is the point of these methods? Should they be marked as `abstractmethod`?

Reply:
They are for subclasses to perform startup/shutdown type activities (as is needed by open telemetry). They need to be defined in the base class because the `Observability` context manager needs to invoke them.

I figured it isn't always going to be necessary, so it seems nice to have the default implementation (no-op) be implemented in the base class rather than requiring subclasses to implement a no-op (which would be needed if marking them as `abstractmethod`).

I don't expect there to be many subclasses, so it's getting close to bikeshedding. Want me to change it?

Reply:
Makes sense. If we're keeping it as non-abstract then maybe `pass` _is_ correct over `...`.